OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -100,7 +101,10 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
             @Override
             public void onPageSelected(int position) {
                 // Disable ViewPager2 scrolling to enable horizontal scrolling to for the WebView (Specific HTML Components).
-                pager2.setUserInputEnabled(pagerAdapter.getUnit(position).getType() != BlockType.DRAG_AND_DROP_V2 && pagerAdapter.getUnit(position).getType() != BlockType.WORD_CLOUD);
+                List<BlockType> horizontalBlocks = Arrays.asList(
+                        BlockType.DRAG_AND_DROP_V2, BlockType.LTI_CONSUMER, BlockType.WORD_CLOUD);
+                pager2.setUserInputEnabled(!horizontalBlocks
+                        .contains(pagerAdapter.getUnit(position).getType()));
```

Comment:
Suggestion:
The app shouldn't disable horizontal scrolling if get `multi_device:false`
```java
                if (pagerAdapter.getUnit(position).isMultiDevice()) {
                    // Disable ViewPager2 scrolling to enable horizontal scrolling to for the WebView (Specific HTML Components).
                    List<BlockType> horizontalBlocks = Arrays.asList(
                            BlockType.DRAG_AND_DROP_V2, BlockType.LTI_CONSUMER, BlockType.WORD_CLOUD);
                    pager2.setUserInputEnabled(!horizontalBlocks
                            .contains(pagerAdapter.getUnit(position).getType()));
                }
```